sklearn/linear_model/tests/test_ridge.py

```diff
@@ -487,6 +487,35 @@ def test_class_weights():
     assert_array_almost_equal(clf.intercept_, clfa.intercept_)
 
 
+def test_class_weight_vs_sample_weight():
```

Comment:
Agreed @jnothman , nothing special to Ridge here, purely a test for classifiers with `class_weight` (constructor) and `sample_weight` (fit) to check equivalency and multiplicative combination. Should be able to work its way into a common test I would think. This test appears in the tree tests (though with a multi-output additional step_ and elsewhere (I think I may have pilfered or altered it from SGD originally).